docs/index.md

```diff
@@ -33,7 +37,8 @@ provider "clumio" {
     role_arn = "..."
     exteral_id = "..."
     session_name = "..."
-    duration_seconds = ...
+    // Assume role duration in seconds.
+    duration_seconds = 1800
```

Comment:
Is this saying that we recommend the assume role timeout to be 30 minutes?

Reply:
Just an example. Some number I have to put. Doesn't matter what number it is.

Reply:
I see, more any specific reason you put it this time around and not before (since it used to say `duration_seconds = ...`?

Reply:
I can leave it that way. Just wanted to make sure it is a number and not a string.